### PR TITLE
Bugfix/ui issues

### PR DESCRIPTION
**Pull request by denlukia**

### CODE REVIEW

File: src/sass/components/_sections.scss [hunk 1]
```diff
@@ -71,30 +82,31 @@
...
+        }
     }
 }
 
 @media screen and (max-width: 768px) {
-    .sections {
-        padding: 55px 0;
-    }
     .section-item,
     .section-item:nth-of-type(1) {
```
Comment (A-Ostrovnyy) [inline src/sass/components/_sections.scss]: If selector empty, remove him.

File: src/templates/partials/_themes.html [hunk 2]
```diff
@@ -1,56 +1,35 @@
...
-    },
-    {
-      themes: 'Як перевчитися і успішно свічнутися?'
-    },
-    {
-      themes: 'Де шукати співтовариство і допомогу у вирішенні питань?'
-    }
-  ]
-  %}
+{% set list = [ { themes: 'Як почати свій шлях в галузі і знайти першу роботу?'
```
Comment (A-Ostrovnyy) [inline src/templates/partials/_themes.html]: Why you collapse this array? Easier to read/edit when it expanded.

File: src/templates/partials/_themes.html [hunk 3]
```diff
@@ -1,56 +1,35 @@
...
       </div>
 
       {% endfor %}
-
     </div>
-
+    <div class="themes__hmm-wrapper">
+      <img
+        class="themes__hmm-gif"
+        src="../../img/Hmm-on-dark.gif"
```
Comment (A-Ostrovnyy) [inline src/templates/partials/_themes.html]: It also works with `src="/img/Hmm-on-dark.gif"` too.

File: src/templates/partials/_organizers.html [hunk 4]
```diff
@@ -57,11 +57,14 @@ <h3 class="organizers__h3">Навіщо ми це робимо?</h3>
           src="/img/kottans-big-citation.svg"
           alt=""
         />
+        <img
+          class="organizers__citation-img--mobile"
+          src="/img/kottans-big-citation-mobile.svg"
+          alt=""
```
Comment (A-Ostrovnyy) [inline src/templates/partials/_organizers.html]: Fill `alt` attribute.

File: src/sass/components/_contacts.sass [hunk 5]
```diff
@@ -1,27 +1,26 @@
...
-	width: 100%
-	background-color: $black
   background: linear-gradient(0deg, rgba(0, 0, 0, 0.75), rgba(0, 0, 0, 0.75)), url(../../img/header-bg-light.jpg)
+  background-size: cover, cover
   
  
   
 .contacts
-	font-weight: 700
+	font-weight: bold
```
Comment (A-Ostrovnyy) [inline src/sass/components/_contacts.sass:8]: Better remove this line.
`font-weight: 700` applied to `html`, and this block inherits this property.

File: src/sass/components/_footer.sass [hunk 6]
```diff
@@ -1,43 +1,48 @@
 .footer-wrapper
-  background-color: $black
+  background: linear-gradient(0deg, rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.9)), url(../../img/header-bg-light.jpg)
```
Comment (A-Ostrovnyy) [inline src/sass/components/_footer.sass]: Change URL `url(/img/header-bg-light.jpg)`

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments above.

File: src/templates/partials/_themes.html (hunk 2 above)
Comment (denlukia) [reply src/templates/partials/_themes.html]: It's not intentional, had to switch VS Code frome Nunjucks to HTML in order to have autocompletion and... Prettier did it's thing ¯\_(ツ)_/¯

File: src/sass/components/_sections.scss (hunk 1 above)
Comment (denlukia) [reply src/sass/components/_sections.scss]: Did it

File: src/templates/partials/_themes.html (hunk 2 above)
Comment (denlukia) [reply src/templates/partials/_themes.html]: Fixed

File: src/templates/partials/_themes.html (hunk 3 above)
Comment (denlukia) [reply src/templates/partials/_themes.html]: Fixed

File: src/templates/partials/_organizers.html (hunk 4 above)
Comment (denlukia) [reply src/templates/partials/_organizers.html]: Fixed

File: src/sass/components/_contacts.sass (hunk 5 above)
Comment (denlukia) [reply src/sass/components/_contacts.sass:8]: I thought 700 is bolder than bold. Removed that line anyway

File: src/sass/components/_footer.sass (hunk 6 above)
Comment (denlukia) [reply src/sass/components/_footer.sass]: Fixed

File: src/sass/components/_contacts.sass (hunk 5 above)
Comment (A-Ostrovnyy) [reply src/sass/components/_contacts.sass:8]: `font-weight: 700` === `font-weight: bold`

**Review by A-Ostrovnyy — APPROVED** [review]

**Pull request merged**